libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -1,5 +1,131 @@
 import { CoinType } from "./slip44";
 
+// All cryptocurrency ids
+// list should be append only. **do not modify existing ids**
+export type CryptoCurrencyId =
```

Comment:
@gre what do you think of this change?

Idea is that `CryptoCurrencyId` was still defined in the `cryptoassets` package, which needed to be imported even when we only needed the IDs. So we better define it in `types-cryptoassets`.